src/App.css

```diff
@@ -84,6 +84,21 @@ body {
   background: #9d6888;
 }
 
+.button-zero {
+  flex: 0 0 calc(50% - 10px)
+}
+
+.fourth-row {
+  position: relative;
+}
+
+#equals {
```

Comment:
This needs width and height specified. It's taken out of the document flow with `position: absolute`